src/libraries/System.Security.Cryptography/ref/System.Security.Cryptography.cs

```diff
@@ -3185,6 +3189,9 @@ public PublicKey(System.Security.Cryptography.Oid oid, System.Security.Cryptogra
         public System.Security.Cryptography.ECDsa? GetECDsaPublicKey() { throw null; }
         [System.Diagnostics.CodeAnalysis.ExperimentalAttribute("SYSLIB5006")]
         [System.Runtime.Versioning.UnsupportedOSPlatformAttribute("browser")]
+        public System.Security.Cryptography.MLDsa? GetMLDsaPublicKey() { throw null; }
```

Comment:
For `MLKem` we have a new constructor as well. Doesn't need to be in this PR - but either all PQC or none should introduce new constructors. We don't... really... need them because they can just export the SPKI and import it in to `PublicKey` using `CreateFromSubjectPublicKeyInfo`, but I feel like it's kind of nice to have it.

Reply:
Ah, right. Because we have one for AsymmetricAlgorithm

Reply:
Adding it here is easy, but the tests are hard to write without the file load test data (which hasn't been done for ML-DSA yet), so I'm going to leave it out to avoid the test debt.